README.md

```diff
@@ -34,11 +34,11 @@ If you would like to contribute to the curriculum, please [sign up here](https:/
 #### Onboarding
 1. [Welcome, Tips, and Rules](/welcome-tips-rules/tech-industry-tips.md)
 1. [How to Learn](/learning-to-learn/learning-to-learn.md)
-1. [Growth Mindset](/growth-mindset/YCLA_LessonPlan_v10.pdf)
 1. [Professionalism](/professionalism/professionalism.md)
-1. [Tech Industry Tips](/welcome-tips-rules/tech-industry-tips.md)
-1. [Roles in Tech](/roles-in-tech/roles-in-tech.md)
 1. [Ergonomics](/ergonomics/ergonomics.md)
+1. [Roles in Tech](/roles-in-tech/roles-in-tech.md)
+1. [Tech Industry Tips](/welcome-tips-rules/tech-industry-tips.md)
+1. [Growth Mindset](/growth-mindset/YCLA_LessonPlan_v10.pdf)
```

Comment:
All points start from number 1 :sweat_smile: Is it meant to be this way?

Reply:
in the actual readme.md, the 1 gets inccremented
thats the syntax